source/interact-data/specify-query.txt

```diff
@@ -484,3 +483,423 @@ as the shown in the following example:
+
+   If you use the {+ruby-driver+} to query on the ``_id`` field,
+   ``find()`` does not internally perform the type conversion.
+
+The ``find()`` method accepts multiple arguments, or an array of arguments.
+{+odm+} interprets each argument or array element as an
+``_id`` value, and returns documents with all the specified ``_id``
+values in an array, as shown in the following example:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-id-query-multiple
+   :end-before: end-id-query-multiple
+   :language: ruby
+   :dedent:
+
+The ``find()`` method exhibits the following behavior:
+
+- If you provide the same ``_id`` value more than once, {+odm+}
+  returns only one document, if one exists.
+
+- {+odm+} does not return documents in an ordered way. Documents might
+  be returned in different order from the order of the provided ``_id``
+  values.
+
+- If any of the ``_id`` values are not found in the database, the result
+  depends on the value of the ``raise_not_found_error`` configuration
+  option.
+  
+  If you set the ``raise_not_found_error`` option to ``true``,
+  ``find()`` raises a ``Mongoid::Errors::DocumentNotFound`` error if any
+  of the ``_id`` values are not found.
+  
+  If you set the ``raise_not_found_error`` option to ``false`` and query
+  for a single ``_id`` value, ``find()`` returns ``nil`` if {+odm+} does
+  not match a document. If you pass multiple ``_id`` values and
+  some or all are not matched, the return value is an array of any documents
+  that match, or an empty array if no documents match.
+
+find() Variations
+-----------------
+
+This section describes methods that are similar to the ``find()`` method
+described in the preceding section.
+
+You can use the ``find_by()`` method to retrieve documents based on the
+provided criteria. If no documents are found, it raises an error or
+returns ``nil`` depending on how you set the ``raise_not_found_error``
+configuration option.
+
+The following code demonstrates how to use the ``find_by()`` method:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-query-findby
+   :end-before: end-query-findby
+   :language: ruby
+   :dedent:
+
+You can use the ``find_or_create_by()`` method to retrieve documents
+based on the provided criteria. If no documents are found, it creates
+and returns an instance that is saved to MongoDB.
+
+The following code demonstrates how to use the ``find_or_create_by()``
+method:
+
+.. literalinclude:: /includes/interact-data/query.rb
+   :start-after: start-query-find-or-create
+   :end-before: end-query-find-or-create
+   :language: ruby
+   :dedent:
+
+You can use the ``find_or_initialize_by()`` method to retrieve documents
+based on the provided criteria. If no documents are found, it returns a
+new one, without persisting it to MongoDB. The syntax for
+``find_or_initialize_by()`` is the same for the ``find_or_create_by()``
+method.
```

Comment:
I _think_ you need an "as" in here: "The syntax for X is the same as [the syntax] for Y."